src/schema/credit_card.js

```diff
@@ -0,0 +1,47 @@
+import {
+  GraphQLInt,
+  GraphQLObjectType,
+  GraphQLString,
+  GraphQLNonNull,
+} from "graphql"
+import { GravityIDFields } from "schema/object_identification"
+
+const CreditCardType = new GraphQLObjectType({
+  name: "CreditCard",
+  fields: () => ({
+    ...GravityIDFields,
+    brand: {
```

Comment:
are any of these required, or can they all be nullable?

Reply:
I believe they can all be nullable

Reply:
This is from Gravity, right? It looks like there's some expectations around these perhaps being non nullable: https://github.com/artsy/gravity/blob/0a4d7b1abc4114f092e1414c0e07c811a9cb7bc9/app/models/domain/credit_card.rb#L31-L34

Reply:
Yep my bad. Looked in `stripe_credit_card` not `credit_card`.